__tests__/integration.test.js

```diff
@@ -94,3 +94,45 @@ describe("GET /api/articles/:article_id", () => {
       });
   });
 });
+
+describe("GET /api/articles", () => {
+  test("responds with a 200 status code", () => {
```

Comment:
This should be combined in your other tests. This is integration testing, not unit testing. Its a ok and expected to combine parts such as the status code and the data that is returned.